ReadMe.md

```diff
@@ -46,6 +46,8 @@ python submission.py submit PATH_TO_FILE [--submission-name NAME] --submit
 uv run submission.py submit PATH_TO_FILE [--submission-name NAME] --submit
 ```
 
+where `submission-name` is an optional human-readable name for the submission.
```

Comment:
Based on just the ReadMe I wasn't sure if this was supposed to be my name, my username, a submission name, etc.